### PR TITLE
Changed travel to traveled in about me

### PR DESCRIPTION
**Pull request by AlignSD**
Simple fix. Your code looks great and I love the responsiveness of the site couldn't find anything to add besides a simple possible spelling error.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h1>Ketan<br>Amar</h1>
       <div class="about-text">
         <h1>About me</h1>
         <p>Hey, people!</p>
-        <p>I believe you know that I am fond of clicking random pictures and travel places that can make anyone's heart beat a
+        <p>I believe you know that I am fond of clicking random pictures and traveling places that can make anyone's heart beat a
```
Comment (stephin007) [inline index.html]: it should be "travel to places" not travelling
please do this change and re raise the PR

**Review by stephin007 — CHANGES_REQUESTED** [review]
pllease check the comment!

File: index.html (hunk 1 above)
Comment (AlignSD) [reply index.html]: will do

**Comment by AlignSD** [comment]
Changed "traveled places" to "travel to places"

edit: formatting

**Review by stephin007 — APPROVED** [review]
LGTM

**Pull request merged**